internal/planner/operations.go

```diff
@@ -34,9 +34,11 @@ var (
 	_ planNode = (*valuesNode)(nil)
 	_ planNode = (*viewNode)(nil)
 	_ planNode = (*lensNode)(nil)
+	_ planNode = (*operationNode)(nil)
```

Comment:
nitpick: `MultiNode` implements/extends `planNode`, so this line is not required.

Reply:
Done